### PR TITLE
chore(postcss-reduce-initial): fix build command

### PR DESCRIPTION
**Pull request by sigveio**
This pull request aims to fix an outdated babel `--ignore` for the `postcss-reduce-initial` acquire script

It was overlooked in #1169

### CODE REVIEW

**Review by ludofischer — APPROVED** [review]

**Comment by sigveio** [comment]
Does the project have any rules/guidelines/common practises when it comes to merging into master - beyond having approval of at least one other with write access? E.g. being handled by certain individuals, not merging own PRs, or that kind of things?

And is there anything in particular you pay attention to? From what I've gathered you always squash multiple commits?

Thought I'd ask before doing, and also in the interest of documenting it at some point =)

**Comment by ludofischer** [comment]
> Does the project have any rules/guidelines/common practises when it comes to merging into master - beyond having approval of at least one other with write access? E.g. being handled by certain individuals, not merging own PRs, or that kind of things?

I don't think there are any special guidelines, as long as somebody else reviews your PRs. @alexander-akait has most permissions on this project. As you can see from the commit history, the people who started the project and contributed most of the code doing the actual transformations have mostly stopped contributing a few years ago, since then there haven't been many committers at all, so I think there never was a need to establish precise guidelines.

**Comment by ludofischer** [comment]
I generally don't merge PRs immediately after approving them to let others have a chance of looking at them.

**Comment by sigveio** [comment]
> I generally don't merge PRs immediately after approving them to let others have a chance of looking at them.

Good to know - thanks :-)

**Pull request merged**
